packages/fxa-settings/src/components/LinkDamaged/en.ftl

```diff
@@ -0,0 +1,12 @@
+## LinkDamaged component
+
+# The user followed a link to reset their password that was received by email
+# but the link was damaged (for example mistyped or broken by the email client).
+password-reset-link-damaged-header = Reset password link damaged
```

Comment:
Is there a reason this needs a new ID? Since these have already been localized and string isn't changing the ID could be left as-is and existing translations could be used.